config/config.properties

```diff
@@ -106,6 +106,47 @@ pravegaservice.dataLog.impl.name=BOOKKEEPER
 # pravegaservice.storage.layout=ROLLING_STORAGE
 # NOTE: ChunkedSegmentStorage is an experimental feature. Its behavior and/or API should be expected to be in flux until fully released.
 
+# Size of chunk in bytes above which it is no longer considered a small object.
+# For small source objects, concat is not used.
+# Valid values: non-negative long
+# Default value: 0
+# storage.concat.size.bytes.min=0
+
+# Size of chunk in bytes above which it is no longer considered for concat.
```

Comment:
Please rephrase this to be refer to the `max` argument.

Reply:
fixed